tools/ci_build/github/linux/docker/Dockerfile.ubuntu_tensorrt

```diff
@@ -2,7 +2,7 @@
 # Label: com.nvidia.cuda.version: 11.6.1
 # Label: com.nvidia.cudnn.version: 8.4.0
 # Ubuntu 20.04
-FROM nvidia/cuda:11.6.1-cudnn8-devel-ubuntu20.04
+FROM nvidia/cuda:11.8.0-cudnn8-devel-ubuntu20.04 AS base
```

Comment:
nitpick: The `AS base` is not needed since this dockerfile does not have another `FROM` statement that builds upon this subimage.

Reply:
removed.